packages/teams-js/src/public/meeting.ts

```diff
@@ -507,10 +529,13 @@ export namespace meeting {
   }
 
   /**
-   * Registers a handler for changes to the live stream.
+   * Registers an event handler for state changes to the live stream.
    *
    * @remarks
    * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
+   * Use {@link requestStartLiveStreaming} or {@link requestStopLiveStreaming} to start/stop a live stream.
+   * This API can only be used in the `sidePanel` frame context.
+   * The `meetingExtensionDefinition.supportsStreaming` field in your app manifest must be `true` to use this API.
```

Comment:
Could you link to appropriate manifest documentation? This is a great detail to include in here.

Reply:
Done!